src/components/map/WebViewMap.tsx

```diff
@@ -49,9 +49,16 @@ export const WebViewMap = ({
   );
 };
 
-const styles = StyleSheet.create({
-  map: {
-    aspectRatio: 16 / 9,
-    width: '100%'
+// this will only be set at the start of the app, so this will be the width of portrait mode
+// needs to be done due to react native 0.63 bug for android
```

Comment:
marking with `TODO:` the next time helps to identify it across the app. even code climate points out comments with `TODO:`, `FIXME:`, ...
this way, it is harder to forget it somewhen.

Reply:
talked with @digorath